lib/src/util/map_util.dart

```diff
@@ -60,6 +61,16 @@ Map getPropsToForward(Map props, {bool omitReactProps: true, bool onlyCopyDomPro
   return propsToForward;
 }
 
+/// Returns a copy of the style map found in [props].
+///
+/// Returns an empty map if [props] or its style map are null.
+Map newStyleFromProps(Map props) {
```

Comment:
IMO, this return signature should match the [type signature of the `style` prop](https://github.com/Workiva/over_react/blob/master/lib/src/component/prop_mixins.dart#L65)...

```dart
Map<String, dynamic> newStyleFromProps(Map props) {
  // ...
}
```